Makefile

```diff
@@ -253,3 +253,20 @@ $(GOLANGCI_LINT): ; $(info installing golangci-lint...)
 .PHONY: lint
 lint: | $(GOLANGCI_LINT) ; $(info  running golangci-lint...) @ ## Run golangci-lint
 	$(GOLANGCI_LINT) run --timeout=10m
+
+$(BIN_DIR):
+	@ mkdir -p $(BIN_DIR)
+
+YQ=$(BIN_DIR)/yq
+YQ_VERSION=v4.44.1
+$(YQ): | $(BIN_DIR); $(info installing yq)
+	@ wget https://github.com/mikefarah/yq/releases/download/$(YQ_VERSION)/yq_linux_amd64 -O $(YQ) && chmod +x $(YQ)
```

Comment:
nit: here you use `wget`, in the chart-update script you are using `curl`. Do we want to align?

Reply:
wget to get (download) file and curl to get response body from server  i dont think there is a particular need to align.

if there will be additional targets in makefile to download files i would envision they use wget.